### ISSUE TITLE
Max sequence length computation error

### ISSUE THREAD

**Issue by DimitrisKatsiros** [tyiannak/deep_audio_features#14]
The code down below **does not compute** the max sequence length. Please check the `length` formula.
https://github.com/tyiannak/deep_audio_features/blob/71e0ba5cfc5fd43613886be807fff8e4205b51f4/utils/load_dataset.py#L115-L121

**Comment by tyiannak** [comment]
thanx @DimitrisKatsiros u'r right, obviously the (config.HOP_LENGTH - config.HOP_LENGTH)) is incorrect, the purpose here was to have the (config.WINDOW_LENGTH - config.HOP_LENGTH)) instead. Or do you mean sth else?

**Comment by DimitrisKatsiros** [comment]
@tyiannak I tried the same code after changing `config.HOP_LENGTH` to `config.WINDOW_LENGTH`. Still produces `0`s as result for wavs lengths.

**Comment by tyiannak** [comment]
What wav files are you giving as input? I've tried in several wavs ranging from 1 to 5 secs and results make sense

**Comment by DimitrisKatsiros** [comment]
@tyiannak wavs from VoxCeleb with duration ~ 1-30 secs. 
I used length formula as: 
```python3
     length = int((duration - 
                   (config.WINDOW_LENGTH - config.HOP_LENGTH)) / \ 
                  (config.HOP_LENGTH) + 1)
```
when `HOP_LENGTH`=`WINDOW_LENGTH/2`=`round(.2 * sr)`, `sr=16000` on my pc.

**Comment by tyiannak** [comment]
both hop_length and window_length are counted in seconds not samples (i changed that in the last PR). 
So you should not use window sizes in seconds, this is handled internally now (no need to provide neither sampling rate nor windows in samples, just give it in seconds in config.py as it is currently in master)

**Comment by DimitrisKatsiros** [comment]
@tyiannak Thank you. Fixed in e26da96. Closing the issue.
